core-blocks/video/edit.js

```diff
@@ -86,8 +101,37 @@ class VideoEdit extends Component {
 						/>
 					</Toolbar>
 				</BlockControls>
+				<InspectorControls>
+					<PanelBody title={ __( 'Video Settings' ) }>
```

Comment:
Does it make sense to label this "Playback Controls", to follow #7322?

Generally, I think "Playback Controls" is more specific than, and thus preferable to, "Video Settings"